components/RemediesList.js

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function RemediesList({ remedies }) {
+  return (
+    <>
+      <h1>Natural Cures</h1>
+      <GridContainer>
+        {remedies.map((remedy) => (
+          <RemedyCard key={remedy.id}>
+            <StyledImage
+              src={remedy.imageUrl}
+              alt={remedy.title}
+              width={250}
+              height={250}
+            />
+            <h2>{remedy.title}</h2>
+            <h3>Symptoms</h3>
+            <ul>
+              {remedy.symptoms.map((symptom, index) => (
+                <li key={index}>{symptom}</li>
+              ))}
+            </ul>
+          </RemedyCard>
+        ))}
+      </GridContainer>
+    </>
+  );
+}
+
+const GridContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-items: center;
+  gap: 10px;
```

Comment:
```suggestion
  gap: 1rem;
```
Using relative units like rem for example helps you building your code responsive from scratch.